source/remoteClient/__init__.py

```diff
@@ -0,0 +1,18 @@
+from .client import RemoteClient
+
+
+def initialize():
+	"""Initialise the remote client."""
+	import globalVars
+	import globalCommands
+
+	globalVars.remoteClient = RemoteClient()
```

Comment:
`globalVars` is scheduled for deprecation. I would currently assign the instance of the client to something like `remoteClient.client`, similar to how there is a global `braille.handler`, etc.